rerun_py/rerun_sdk/rerun/log/experimental/blueprint.py

```diff
@@ -0,0 +1,160 @@
+        try to pick a unique one for each application that uses the Rerun SDK.
+
+        For example, if you have one application doing object detection
+        and another doing camera calibration, you could have
+        `rerun.init("object_detector")` and `rerun.init("calibrator")`.
+    blueprint_id : Optional[str]
+        Set the blueprint ID that this process is logging to, as a UUIDv4.
+
+        The default blueprint_id is based on `multiprocessing.current_process().authkey`
+        which means that all processes spawned with `multiprocessing`
+        will have the same default recording_id.
+
+        If you are not using `multiprocessing` and still want several different Python
+        processes to log to the same Rerun instance (and be part of the same blueprint),
+        you will need to manually assign them all the same blueprint_id.
+        Any random UUIDv4 will work, or copy the recording id for the parent process.
+    make_default : bool
+        If true (_not_ the default), the newly initialized blueprint will replace the current
+        active one (if any) in the global scope.
+    make_thread_default : bool
+        If true (_not_ the default), the newly initialized blueprint will replace the current
+        active one (if any) in the thread-local scope.
+    spawn : bool
+        Spawn a Rerun Viewer and stream logging data to it.
+        Short for calling `spawn` separately.
+        If you don't call this, log events will be buffered indefinitely until
+        you call either `connect`, `show`, or `save`
+    add_to_app_default_blueprint
+        Should the blueprint append to the existing app-default blueprint instead instead of creating a new one.
+    default_enabled
+        Should Rerun logging be on by default?
+        Can overridden with the RERUN env-var, e.g. `RERUN=on` or `RERUN=off`.
+
+    Returns
+    -------
+    RecordingStream
+        A handle to the [`rerun.RecordingStream`][]. Use it to log data to Rerun.
+
+    """
+
+    blueprint_id = application_id if add_to_app_default_blueprint else blueprint_id
+
+    blueprint = RecordingStream(
+        bindings.new_blueprint(
+            application_id=application_id,
+            blueprint_id=blueprint_id,
+            make_default=make_default,
+            make_thread_default=make_thread_default,
+            default_enabled=default_enabled,
+        )
+    )
+
+    if spawn:
+        from rerun.sinks import spawn as _spawn
+
+        _spawn(recording=blueprint)
+
+    return blueprint
+
+
+def add_space_view(
+    name: str,
+    space_path: str,
+    entity_paths: List[str],
+    blueprint: Optional[RecordingStream] = None,
+) -> None:
+    """
+    Add a new space view to the blueprint.
+
+    Parameters
+    ----------
+    name : str
+        The name of the space view.
+    space_path : str
+        The anchor point of the space view.
+    entity_paths : List[str]
+        The entities to be shown in the space view.
```

Comment:
Instinctively as a user I want to just say "Create a space-view with these objects", e.g. `add_space_view(["rgb", "depth"])`. Name feels like it could be optional, and `space_path` is just under-explained. What does it mean?

Perhaps we should just make all arguments named for now to make it easier to move/remove them later?

```suggestion
def add_space_view(
    *,
    name: str,
    space_path: str,
    entity_paths: List[str],
    blueprint: Optional[RecordingStream] = None,
) -> None:
    """
    Add a new space view to the blueprint.

    Parameters
    ----------
    name : str
        The name of the space view.
    space_path : str
        The anchor point of the space view.
    entity_paths : List[str]
        The entities to be shown in the space view.
```